zebra/if_netlink.c

```diff
@@ -468,10 +466,7 @@ netlink_interface_addr (struct sockaddr_nl *snl, struct nlmsghdr *h,
   ifa = NLMSG_DATA (h);
 
   if (ifa->ifa_family != AF_INET
-#ifdef HAVE_IPV6
-      && ifa->ifa_family != AF_INET6
-#endif /* HAVE_IPV6 */
-    )
+      && ifa->ifa_family != AF_INET6)
     return 0;
```

Comment:
Please, join this 2 lines:
if (ifa->ifa_family != AF_INET && ifa->ifa_family != AF_INET6)